ver/us/splat.yaml

```diff
@@ -254,7 +254,8 @@ segments:
     - [0x4A2A0, c, os/osFlash]
     - [0x4AC90, pm_effect_loads, effect_loads]
     - [0x4DF30, c, battle/battle]
-    - [0x4E5A0, data, 1000_len_60]
+    - [0x4E5A0, bin, n_aspMain_text]
+    - [0x4F200, data, 1000_len_60]
```

Comment:
This file should be renamed to just 4F200 (4F200.data.s) since it's not at offset 1000